core/templates/domain/user/user-profile.model.ts

```diff
@@ -43,7 +45,7 @@ export class UserProfile {
     public usernameOfViewedProfile: string,
     public userBio: string,
     public subjectInterests: string[],
-    public firstContributionMsec: number,
+    public firstContributionMsec: number | null,
```

Comment:
```suggestion
    public firstContributionMsec!: number,
```

Reply:
`null` is assigned as a value later so it used a  type, `!` is for bypassing `strictPropertyInitialization` error, we need not to use it here because it's a model file and we don't initialize properties here.